writeletter.rs

```diff
@@ -0,0 +1,65 @@
+fn main() {
+    println!("Enter the first name of the person you want to write to:");
+
+    let mut first_name = String::new();
+    std::io::stdin().read_line(&mut first_name).unwrap();
+    first_name.pop();
+
+    println!("Dear {},", first_name);
+    println!("  I'm writing this letter because I miss you.");
+    println!("Lots of things have happened since I started learning Rust.");
+    println!("I started off with Python, and now I'm learning a completely new language.");
+
+    println!("\nEnter the name of another close friend:");
+
+    let mut friend_name = String::new();
+    std::io::stdin().read_line(&mut friend_name).unwrap();
+    friend_name.pop();
+
+    println!("Have you seen {} lately?", friend_name);
+
+    println!("\nEnter m if the friend is male or f if the friend is female:");
+    let mut friend_sex = String::new();
+    std::io::stdin().read_line(&mut friend_sex).unwrap();
+    friend_sex.pop();
+
+    match friend_sex.as_str() {
+        "m" => {
+            println!("If you see {} please ask him to call me.", friend_name);
+        }
+        "f" => {
+            println!("If you see {} please ask her to call me.", friend_name);
+        }
+        _ => {
+            println!("If you see {} please ask them to call me.", friend_name);
+        }
+    }
+
+    println!("\nEnter the age of your friend:");
+    let mut age = String::new();
+    std::io::stdin().read_line(&mut age).unwrap();
+    let number: i32 = age.trim().parse().unwrap();
+    age.pop();
```

Comment:
What is this for?

Reply:
I tried to get the user input through let mut age.
Then I had to change it into a i32 form to compare the numbers for the if-else statements.
I put in age.pop() so that the enter blank would not be included in the input.

Reply:
Why do you need to remove the enter (`\n`) from `age`? After calling `age.pop()`, the program never uses `age`.

Reply:
Understood. I will change the variable.